ext/spl/tests/iterable_to_array.phpt

```diff
@@ -0,0 +1,83 @@
+--TEST--
+SPL: iterable_to_array() test
+--FILE--
+<?php
+
+$array = ['a' => 1, 'b' => 2, 'c' => 3];
+$iterator = new ArrayIterator($array);
+$generator = function () use ($array) { yield 'a' => 1; yield 'a' => 2; yield 'a' => 3; };
```

Comment:
The `use ($array)` seems unnecessary.

Reply:
Right! I originally had `yield from $array;` there. :wink: